README.md

```diff
@@ -65,32 +65,36 @@ You can add `cwd=elmSource` to the loader:
 ```js
 var elmSource = __dirname + '/elm/path/in/project'
   ...
-  loader: 'elm-webpack?cwd=' + elmSource
+  use: 'elm-webpack-loader?cwd=' + elmSource
   ...
 ```
 
 You can use this to specify a custom location within your project for your elm files. Note, this
-will cause the compiler to look for **all** elm source files in the specified directory. This approach is recommended as it allows the compile to watch elm-package.json as well as every file in the source directories.
+will cause the compiler to look for **all** elm source files in the specified directory. This
+approach is recommended as it allows the compile to watch elm-package.json as well as every file
+in the source directories.
```

Comment:
Nothing apart from the formatting, simply preventing the line length from exceeding 100 chars.